.ado/templates/apple-xcode-select.yml

```diff
@@ -3,6 +3,6 @@
 #
 steps:
   - bash: |
-      sudo xcode-select --switch '/Applications/Xcode_11.4.1.app'
-    displayName: 'Switch to XCode 11.4.1'
+      sudo xcode-select --switch '/Applications/Xcode_12.app'
+    displayName: 'Switch to XCode 12'
```

Comment:
I should probably revert this, but also worth noting Azure Pipelines is dropping support for Xcode 11.4 in November... so might as well keep it in?

Reply:
Anywhere else you need to update this?

Reply:
Nope! We already moved to XCode 12 in FluentUI Apple and in Office. Just here should be fine